packages/sanity/src/components/image/Image.tsx

```diff
@@ -42,7 +43,7 @@ export const Image = forwardRef(function Image(
   ref: React.ForwardedRef<HTMLImageElement>
 ) {
   const {dpr, fit, height, source, width, ...restProps} = props
-  const client = useClient()
+  const client = useClient(DEFAULT_STUDIO_CLIENT_OPTIONS)
```

Comment:
Did you consider `useDefaultStudioClient()`? Would reduce number of imports if nothing else.

Reply:
I did, but from a teaching perspective I find it clearer that we use the same hooks that other people would use. We can always refactor this later if it becomes annoying though.